src/components/features/MessageBox/BottomButtons/index.tsx

```diff
@@ -12,14 +12,14 @@ const BottomButtons = ({
 	setShowCheckedMessages,
 }: BottomButtonsProps) => {
 	const queryClient = useQueryClient();
-	const { mutate: openMessagesMutate } = useMutation(() => openMessages(checkMessages), {
+	const { mutate: updateOpenMessagesMutate } = useMutation(() => updateOpenMessages(checkMessages), {
 		onSuccess: () => {
 			queryClient.invalidateQueries('getMessages');
 		},
 	});
 
 	const onClickMakeFruitButton = () => {
-		openMessagesMutate();
+		updateOpenMessagesMutate();
```

Comment:
```suggestion
		updateOpenMessagesMutate.mutate(checkMessages);
```

호출하는 시점에서 mutate 에 필요한 인수를 전달하는 방법도 있더라구요 ~ 참고하시면 좋을 것 같습니다 !